content/docs/user-guide/remote-objects.md

```diff
@@ -0,0 +1,120 @@
+# Working with Remote Objects and Projects
+
+[MLEM objects] (models, data, etc.) can live in different locations such as Git
+repositories, cloud object storage, local directories, etc.
+
+<admon type="note">
+
+In this page we will work with model objects in Git repos, but the same
+operations apply to any [object type] and location.
+
+</admon>
+
+[mlem objects]: /doc/user-guide/basic-concepts#mlem-objects
+[object type]: /doc/user-guide/basic-concepts#mlem-object-types
+
+## Listing objects
+
+You can list MLEM objects inside a remote MLEM projects (Git repo) with
+`mlem list`. There's no need to clone the repo.
+
+```cli
+$ mlem list \
+       https://github.com/iterative/example-mlem-get-started \
+       --rev simple
+Deployments:
+ - myservice
+Models:
+ - rf
+Envs:
+ - staging
+```
+
+<admon type="note">
+
+A MLEM project is required as target for `mlem list`. The other operations
+(below) work with loose MLEM objects as well.
+
+</admon>
+
+## Loading objects (Python)
+
+You can load objects from remote locations inside Python code with
+`mlem.api.load()` by using an object name and its URL.
+
+```py
+from mlem.api import load
+
+model = load(
+    "rf",
+    project="https://github.com/iterative/example-mlem-get-started",
+    rev="simple"
+)
+```
+
+This fetches the `rf` model [form branch `simple`] of the
+`example-mlem-get-started` repo and loads it to memory.
+
+[form branch `simple`]:
+  https://github.com/iterative/example-mlem-get-started/tree/simple/.mlem/model
+
+## Downloading objects
+
+You can download MLEM object files to the local environment in with `mlem clone`
+(CLI).
+
+```cli
+$ mlem clone rf \
+  --project https://github.com/iterative/example-mlem-get-started \
+  --rev simple \
+  ml_model
+⏳️ Loading meta from https://github.com/iterative/example-mlem-get-started/tree/simple/.mlem/model/rf.mlem
+🐏 Cloning https://github.com/iterative/example-mlem-get-started/tree/simple/.mlem/model/rf.mlem
+💾 Saving model to .mlem/model/ml_model.mlem
+```
+
+This places the `rf` model [form branch `simple`] of the
+`example-mlem-get-started` repo, renames it to `ml_model`, and places it in the
+`.mlem/model` directory.
+
+## Cloud storage
+
+It's also possible to (down)load loose MLEM objects stored in any cloud platform
+[supported by `fsspec`], e.g. Amazon S3. To do so, provide the file system
+protocol & path as target/URL, e.g. `s3://<bucket>/`
+
+<admon type="tip">
+
+Loose objects are typically stored this way because they do not require
+[versioning].
+
+</admon>
+
+```cli
+$ mlem init s3://example-mlem-get-started  # Requires S3 auth.
```

Comment:
Why do you have to `mlem init` here? @mike0sv @aguschin thanks

Reply:
I see. But this section is framed as applying to loose MLEM objects (not in a MLEM project) so it is indeed confusing to see `mlem init`.

- [ ] There should probably be a separate section to explain that you can initialize MLEM projects in remote locations and give some motivation (why would I do that). The cmd ref. also should be much more emphatic about this.

For now I removed that line.